pyqtgraph/parametertree/parameterTypes.py

```diff
@@ -146,8 +146,8 @@ def makeWidget(self):
             w = QtWidgets.QLineEdit()
             w.setStyleSheet('border: 0px')
             w.sigChanged = w.editingFinished
-            w.value = lambda: asUnicode(w.text())
-            w.setValue = lambda v: w.setText(asUnicode(v))
+            w.value = w.text
+            w.setValue = w.setText
```

Comment:
@j9ac9k I'm almost inclined to leave `setValue` as casting to string. While you can never get a non-string from `text()`, you can certainly set with e.g. an `int` and this would safely convert. What are your thoughts? It plays into #1919 merging.

Reply:
The new definition of `_interpretValue()` in 1919 should handle a call to `setValue(10)`, but this would also work with `item.widget.setValue(10)`

Reply:
Eh, the widget should inherently function as a text setter, so I'm fine with leaving it as `w.setValue = w.setText`.

Reply:
Feel free to correct things in #1919 as you see fit, again, sorry for the trouble!

Our PR count is low enough we're getting more aggressive about small changes in lots of the library... this is going to keep happening until we effectively zero out the PR queue :(   again, sorry for the headache!